mypy/checker.py

```diff
@@ -4194,6 +4171,39 @@ def push_type_map(self, type_map: 'TypeMap') -> None:
             for expr, type in type_map.items():
                 self.binder.put(expr, type)
 
+    def infer_issubclass_maps_literal_type(self, node: CallExpr,
+                                           expr: Expression,
+                                           type_map: Dict[Expression, Type]
+                                           ) -> Tuple[TypeMap, TypeMap]:
+        """Infer type maps for issubclass calls on literal type."""
```

Comment:
This function has nothing to do with literal types. It just infers type restrictions for an expression (when possible) that appears in an `issubclass()` call.